scripts/generate-releases.sh

```diff
@@ -6,6 +6,32 @@ SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 ROOT_DIR="$(realpath "$SCRIPT_DIR/..")"
 source "$SCRIPT_DIR/helpers.sh"
 
+YQ_VERSION="v4.44.3"
+YQ="${ROOT_DIR}/.bin/yq-${YQ_VERSION}"
+
+# Make sure appropriate YQ is available. There are incompatibilities between versions so we use a version which is
+# known to produce good results.
+ensure_yq() {
+    if [[ -x "${YQ}" ]]; then
+        return 0
+    fi
+    local kernel_name
+    local os_name
+    local arch
+    kernel_name="$(uname -s)"
+    os_name="$(uname | tr '[:upper:]' '[:lower:]')"
+    arch="$(go env GOARCH)"
+    local -r url="https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_${os_name}_${arch}"
+
+    rm -f "${YQ}" # In case it's a stale partial download.
+    mkdir -p "$(dirname "${YQ}")"
+    echo "Fetching ${url}..." >&2
+    curl --silent --fail --location --retry 3 --output "${YQ}" "${url}"
+    [[ "${kernel_name}" != "Darwin" ]] || xattr -c "${YQ}"
```

Comment:
If you will be changing this file again, here's a suggestion because I keep stumbling over this line.
```suggestion
    if [[ "${kernel_name}" == "Darwin" ]]; then
        xattr -c "${YQ}"
    fi
```

Reply:
https://github.com/stackrox/operator-index/pull/189